src/main/resources/application.yaml

```diff
@@ -45,19 +45,19 @@ stargate:
       service:
         # Open AI embedding service configuration
         openai:
-          enabled: false
+          enabled: true
           url: https://api.openai.com/v1/
           api-key: YOUR_API_KEY
 
         # Hugging face embedding service configuration
         huggingface:
-          enabled: false
-          url: https://api-inference.huggingface.co
+          enabled: true
+          url: https://api-inference.huggingface.co/pipeline/feature-extraction
```

Comment:
Not sure on what is the default url for hugging-face. Anyway property that can be override but need to be fixed.